workloads/librispeech/librispeech_jax/input_pipeline.py

```diff
@@ -0,0 +1,55 @@
+"""Data pipeline for LibriSpeech dataset modified from https://github.com/lsari/librispeech_100/blob/main/dataset.py."""
```

Comment:
it looks like this file is 90% the same as the pytorch version; is it possible to move them into a common `workloads/librispeech/input_pipeline.py`, and then subclass one here where the only thing it does is:
- override pad_collate
- call the super class impl
- do the final few `np.array` lines

and then another subclass in the pytorch/ dir which does the same thing but just calls `return torch.utils.data.dataloader.default_collate(batch)`